### PR TITLE
add new policy for getInstitutions and filter on product

### PR DESCRIPTION
**Pull request by KevinSi96**
### List of changes

added new filter on product to the getInstitutionsUsingGet API of the External Api Service

### Motivation and context

need to filter the list of institution based on the onboarded product

### Type of changes

- [ ] Add new resources
- [x] Update configuration to existing resources
- [ ] Remove existing resources

### Env to apply

- [x] DEV
- [x] UAT
- [x] PROD

### Does this introduce a change to production resources with possible user impact?

- [x] Yes, users may be impacted applying this change
- [ ] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [x] No

### Other information

---

### CODE REVIEW

**Review by antonioT90 — DISMISSED** [review]

File: src/core/apim.tf [hunk 1]
```diff
@@ -372,7 +372,9 @@ module "apim_external_api_ms_v1" {
   api_operation_policies = [
     {
       operation_id = "getInstitutionsUsingGET"
-      xml_content  = file("./api/jwt_auth_op_policy_user_group.xml")
+      xml_content = templatefile("./api/party_process/getInstitution_op_policy.xml.tpl", {
+        CDN_STORAGE_URL = "https://${module.checkout_cdn.storage_primary_web_host}"
+      })
     }
   ]
 }
```
Comment (diegolagospagopa) [inline src/core/apim.tf:380]: Please Kevin, configure your Editor to put an empty line in each file

**Review by diegolagospagopa — CHANGES_REQUESTED** [review]

**Review by antonioT90 — APPROVED** [review]

**Review by rodoviti — APPROVED** [review]
ACK

File: src/core/apim.tf (hunk 1 above)
Comment (rodoviti) [inline src/core/apim.tf]:
```suggestion
}

```

**Review by rodoviti — CHANGES_REQUESTED** [review]
add newline

**Review by rodoviti — APPROVED** [review]
ACK

**Review by andrea-putzu — APPROVED** [review]

**Review by diegolagospagopa — APPROVED** [review]

**Pull request merged**